node/node.go

```diff
@@ -425,6 +426,10 @@ func (node *Node) validateNodeMessage(ctx context.Context, payload []byte) (
 					utils.Logger().Debug().Uint64("receivedNum", block.NumberU64()).
 						Uint64("currentNum", curBeaconHeight).Msg("beacon block sync message rejected")
 					return nil, 0, errors.New("beacon block height smaller than current height beyond tolerance")
+				} else if block.NumberU64()-beaconBlockHeightTolerance > curBeaconHeight {
```

Comment:
This is risky. If a node is lagging behind, it will reject all legit beacon block messages, potentially preventing a legit beacon block broadcast to the network.

Reply:
resolved offline.